samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_eu.properties

```diff
@@ -514,9 +514,9 @@ options=Aukerak
 type=Mota
 type_onebyone=Galderak banan-banan eginak
 random_draw_type=Ausazko bista hemendik\:
-random_draw_msg=Parte honetako galderak ausaz sortzen dira galdera-biltzaile batetik. Egin klik Ebaluazioaren aurrebistan adibide bat ikusteko. 
-random_draw_msg_published=Parte honetako galderak galderen gordailuak sortu ditu, <b>{0}</b>, on <b>{1}</b> at <b>{2}</b>. Parte honetako galderak editatzeko, egin klik hemen\: <b>Editatu galderak</b>.
-random_draw_msg_no_date=Parte honetako galderak ausaz sortzen dira galdera gordailu batetik. Egin klik \u2018Aurreko azterketa\u2019 dioen tokian lagin bat ausaz ateratzeko.
+random_draw_msg=Parte honetako galderak ausaz sortzen dira galdera-biltzaile batetik . Egin klik Ebaluazioaren aurrebistan adibide bat ikusteko. 
+random_draw_msg_published=Parte honetako galderak galderen gordailuak sortu ditu , <b>{0}</b>, on <b>{1}</b> at <b>{2}</b>. Parte honetako galderak editatzeko, egin klik hemen\: <b>Editatu galderak</b>.
+random_draw_msg_no_date=Parte honetako galderak ausaz sortzen dira galdera gordailu batetik . Egin klik \u2018Aurreko azterketa\u2019 dioen tokian lagin bat ausaz ateratzeko.
```

Comment:
@pablosanroman Could you please add the changes to these lines?

I copy here the modified Spanish ones:
random_draw_msg=Las preguntas de esta parte son generadas aleatoriamente a partir de una(s) bater\u00eda(s) de preguntas, <b>{0}</b>, en <b>{1}</b> a <b>{2}</b>. Para obtener los \u00faltimos cambios pulsa en <b>Actualizar preguntas</b> o pulsa en <b>Editar</b> de la parte y finalmente en <b>Guardar</b>. Pulsando en <b>Vista previa</b> o en <b>Publicar</b> se actualizar\u00e1 autom\u00e1ticamente el grupo de preguntas al igual que al exportar un examen.
random_draw_msg_published=Las preguntas de esta parte se generaron desde la(s) bater\u00eda(s) de preguntas, <b>{0}</b>, en <b>{1}</b> en <b>{2}</b>. Para editar las preguntas en esta parte, haga clic en <b>Editar preguntas</b>.
random_draw_msg_no_date=Las preguntas de esta parte son generadas aleatoriamente a partir de una(s) bater\u00eda(s) de preguntas. Haga clic en Vista previa para ver un ejemplo.

Reply:
Hi @bgarciaentornos , here they are: 

random_draw_msg=Parte honetako galderak ausaz jasota daude biltzaile batetik edo hainbatetik,  <b>{0}</b>, on <b>{1}</b> at <b>{2}</b>. Sakatu "Aurrebista" adibide bat ikusteko. Azken aldaketak lortzeko sakatu "Eguneratu galderak" edo sakatu partearen "Editatu" eta gero "Gorde". "Aurrebista" edo "Argitaratu" sakatuz gero, automatikoki eguneratuko da galdera-multzoa, azterketa esportatzean bezalaxe.

random_draw_msg_published=Parte honetako galderak {0} biltzaileetatik jasota daude: <b>{0}</b>, on <b>{1}</b> at <b>{2}</b>. Parte honetako galderak editatzeko sakatu "Editatu galderak" botoia.

random_draw_msg_no_date=Parte honetako galderak ausaz jasota daude biltzaile batetik edo hainbatetik. Sakatu "Aurrebista" adibide bat ikusteko.